src/bluebottle.cuh

```diff
@@ -250,4 +250,7 @@ extern __constant__ real _node_p[NNODES];
  *  phi location of nodes
  */
 
+extern __constant__ int _s_mm[64];
```

Comment:
Can you leave a comment here about what these coeffs are? Also, what's the reason for initializing this to a static size of 64?